webapp/views.py

```diff
@@ -41,16 +45,26 @@ def download_thank_you(category):
     except IOError:
         mirrors = []
 
+    # Check country code
+    country_code = "NO_COUNTRY_CODE"
+    ip_location = ip_reader.get(flask.request.remote_addr)
+    mirror_list = []
+
+    if ip_location:
+        country_code = ip_location["country"]["iso_code"]
+
     mirror_list = [
```

Comment:
If `ip_location` is required otherwise you get an empty list. You could indent this assignment so its only run if `ip_location` is not set.